netmaster/master/netmaster.go

```diff
@@ -36,6 +36,13 @@ const (
 	defaultSkyDNSImage  = "skynetservices/skydns:latest"
 )
 
+// Run Time config of netmaster
+type nmRunTimeConf struct {
+	clusterMode string
+}
+
+var masterRTCfg nmRunTimeConf
```

Comment:
Hmm... got it; is this read always after netmaster start? If yes, we are covered.